chaosazure/__init__.py

```diff
@@ -81,7 +115,9 @@ def __get_credentials(creds):
     credentials = ServicePrincipalCredentials(
         client_id=creds['azure_client_id'],
         secret=creds['azure_client_secret'],
-        tenant=creds['azure_tenant_id']
+        tenant=creds['azure_tenant_id'],
+        cloud_environment=eval("msrestazure.azure_cloud.{}"
```

Comment:
The `eval()` is not a good idea here. Too dangerous. Could you please create a function that returns the appropriate value?

Reply:
updated with `__get_cloud_env_by_name()`